src/workflows/recipes/components/RevisionNotice.js

```diff
@@ -3,46 +3,57 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
-import { REVISION_DRAFT, REVISION_OUTDATED } from 'console/state/constants';
 import {
-  isRevisionPendingApproval,
-  getRevisionDraftStatus,
-} from 'console/state/revisions/selectors';
+  REVISION_APPROVED,
+  REVISION_DRAFT,
+  REVISION_LIVE,
+  REVISION_PENDING_APPROVAL,
+  REVISION_OUTDATED,
+  REVISION_REJECTED,
+} from 'console/state/constants';
+import { getRevisionDraftStatus, getRevisionStatus } from 'console/state/revisions/selectors';
 
 @connect((state, { revision }) => ({
-  enabled: revision.getIn(['recipe', 'enabled'], false),
-  isPendingApproval: isRevisionPendingApproval(state, revision.get('id')),
-  status: getRevisionDraftStatus(state, revision.get('id')),
+  draftStatus: getRevisionDraftStatus(state, revision.get('id')),
+  status: getRevisionStatus(state, revision.get('id')),
 }))
-class RevisionNotice extends React.PureComponent {
+class RevisionNotice extends React.Component {
   static propTypes = {
-    enabled: PropTypes.bool.isRequired,
-    isPendingApproval: PropTypes.bool.isRequired,
+    draftStatus: PropTypes.string,
     status: PropTypes.string,
   };
 
-  static defaultProps = {
-    status: null,
-  };
+  // THIS SHOULDN'T BE NECESSARY
+  // https://github.com/mozilla/delivery-console/issues/680
+  shouldComponentUpdate(nextProps) {
+    const { draftStatus, status } = this.props;
+    return nextProps.draftStatus !== draftStatus || nextProps.status !== status;
+  }
 
   render() {
-    const { enabled, isPendingApproval, status } = this.props;
+    const { draftStatus, status } = this.props;
 
     let message;
     let type;
 
-    if (isPendingApproval) {
+    if (status === REVISION_PENDING_APPROVAL) {
       message = 'This is pending approval.';
```

Comment:
I think we should make these messages more consistent:
`You are viewing a draft that is pending approval.`